Code/GraphMol/FileParsers/CMakeLists.txt

```diff
@@ -4,14 +4,14 @@ add_definitions(-DRDKIT_FILEPARSERS_BUILD)
 
 if(RDK_USE_BOOST_IOSTREAMS)
 if(WIN32)
-  find_package(Boost ${RDK_BOOST_VERSION} COMPONENTS system iostreams REQUIRED)
+  find_package(Boost ${RDK_BOOST_VERSION} COMPONENTS system iostreams filesystem REQUIRED)
```

Comment:
I generally try to be very careful about adding additional binary dependencies to the RDKit, this includes new boost libraries. New binary dependencies can end up making things difficult for people who are using the rdkit in their own software (not to mention that they can make *our* lives difficult as developers).

I'd prefer not to add boost::filesystem unless we have a really good reason for it.
It looks like you're just using it to handle a filename/path, which the rest of the RDKit successfully uses with standard string manipulations. Did I miss some important, compelling reason that we should start using this library?